web/themes/custom/accessibility_theme/src/patterns/components/heading/heading.json

```diff
@@ -0,0 +1,6 @@
+{
+  "heading_level": "1",
```

Comment:
Let's remove the value of 1 from here.  Since we are setting a default of 2 if a value is not present, we don't need to specify another default value here.